### DISCUSSION TITLE
Anomaly Detection using mix of good and anomalous images

### DISCUSSION THREAD

**Discussion by ramakrse** [open-edge-platform/anomalib#1228] [Q&A]
Looking for help in dataset configuration, when I have two folders
1. Good collection of images (Train)
2. Good as well as Anomalous images (Test)
The task is the detection of anomalies in test dataset and visualization of anomaly heatmap visualization.

**Comment by samet-akcay** [comment]
Hi @ramakrse, are the good and anomalous images in the same folder in your test set? Can you add the tree structure of your dataset please

**Comment by ramakrse** [comment]
@samet-akcay ,
Unable to add structure to test, as of now, I didn't have ground truth.

Task to identify anomaly detection, not classification. As of now, metrics are not applicable.
